tests/Application/src/Factory/LegacyBookFactory.php

```diff
@@ -14,9 +14,9 @@
 namespace App\Factory;
 
 use Sylius\Component\Resource\Exception\UnexpectedTypeException;
-use Sylius\Component\Resource\Translation\Provider\TranslationLocaleProviderInterface;
 use Sylius\Resource\Factory\FactoryInterface;
 use Sylius\Resource\Model\TranslatableInterface;
+use Sylius\Resource\Translation\Provider\TranslationLocaleProviderInterface;
```

Comment:
Maybe not a good idea here, cause it's a legacy factory.

Reply:
Hum there are some other changes to do on this legacy part on tests... I should make another PR to fix them all.